samples/test/create-dataset-tabular-bigquery.test.js

```diff
@@ -18,41 +18,37 @@
 
 const path = require('path');
 const {assert} = require('chai');
-const {after, describe, it} = require('mocha');
+const {after, it} = require('mocha');
 
 const uuid = require('uuid').v4;
 const cp = require('child_process');
 const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
 const cwd = path.join(__dirname, '..');
 
 const datasetDisplayName = `temp_create_dataset_tables_bigquery_test_${uuid()}`;
-const bigquerySourceUri =
-  'bq://prj-ucaip-tutorials.bigquery_dataset.walmart_triptrain_train';
+const bigquerySourceUri = 'bq://ucaip-sample-tests.table_test.all_bq_types';
 const project = process.env.CAIP_PROJECT_ID;
 const location = process.env.LOCATION;
 
 let datasetId;
 
-// Refs: https://github.com/googleapis/nodejs-ai-platform/issues/187
-describe.skip('AI platform create dataset tabular bigquery', () => {
-  it('should create a new bigquery tabular dataset in the parent resource', async () => {
-    const stdout = execSync(
-      `node ./create-dataset-tabular-bigquery.js ${datasetDisplayName} \
+it('should create a new bigquery tabular dataset in the parent resource', async () => {
```

Comment:
We probably want to keep the `describe` block, as it helps separate out scope of the tests, and makes finding failures in the logs a little easier.  You of course can just remove the `.skip` part :)